src/components/interaction-area/interaction-area.utils.js

```diff
@@ -85,6 +89,42 @@ export const setInteractionAreasContainer = (videojs, newInteractionAreasContain
   }
 };
 
+const getInteractionAreaElementById = (interactionAreasContainer, item, index) => interactionAreasContainer.querySelector(`[data-id=${getInteractionAreaItemId(item, index)}]`);
+
+
+export const updateInteractionAreasItem = (videojs, playerOptions, interactionAreasData, previousInteractionAreasData, onClick) => {
```

Comment:
I'd add a comment above each section of such functions explaining what's being done